src/i18n/Localizable.xcstrings

```diff
@@ -23454,7 +23444,7 @@
       }
     },
     "Suggest" : {
-      "comment" : "is this one used as verb or noun?",
+      "comment" : "is this an action invoked by user? for example, \"suggest\" to somebody. or is this a sort of suggestion given by WebSSH?  \nthe former would use gerund, and noun for the latter.",
```

Comment:
An action invoked by the user. The "Suggest" word is a button label.

[TEXTAREA] (Suggest button)